FrostySdk/Managers/Entries/ResAssetEntry.cs

```diff
@@ -77,6 +77,7 @@ public enum ResourceType : uint
         ShaderBlockDepotResource = 0x73312045,
         CompiledBytecode = 0xE2B02F7,
         ShaderBlockDepot = 0xD8F5DAAF,
+        EAClothColliderSetAssetData = 0x8EF3938D,
```

Comment:
small note, but this should be ```ClothColliderSetAssetData``` according to[ frosty v2](https://github.com/FrostyToolsuite/FrostyToolsuite/blob/f9f79e5db89f8257cf93fbac1188c23dd6018011/FrostySdk/Managers/Entries/ResourceType.cs#L18C3-L18C28)
moreover, running the fnv1 hash function of "clothcollidersetassetdata" gives 0x8EF3938D

```suggestion
        ClothColliderSetAssetData = 0x8EF3938D,
```

Reply:
Since 1.0.7 was missing more ResourceTypes, I just decided to take them all from FrostyV2. | https://github.com/wavebend/FrostyToolsuite/pull/2/commits/5558787a17e83ef34eb46b38bb372b2cf4434a12

Reply:
sounds good